scripts/cmake-presets/ci-centos-rocm.json

```diff
@@ -34,59 +33,44 @@
       }
     },
     {
-      "name": "asan",
+      "name": "asan-orange",
       "description": "Build with address sanitizer flags",
-      "inherits": ["base"],
+      "inherits": ["base", ".reldeb"],
       "cacheVariables": {
         "CELERITAS_USE_HIP": {"type": "BOOL", "value": "OFF"},
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
         "CMAKE_CXX_FLAGS_RELWITHDEBINFO": "-O2 -g -DNDEBUG -fsanitize=address -fno-omit-frame-pointer"
       }
     },
     {
-      "name": "debug",
-      "description": "Build debug with HIP/ROCM for AMD",
-      "inherits": "base"
-    },
-    {
-      "name": "minitest",
-      "description": "Build without any dependencies but JSON/Googletest",
-      "inherits": "base",
+      "name": "debug-orange",
```

Comment:
```suggestion
      "_comment1": "Since debug + HIP doesn't work (too many registers required) I'm making this one the 'minimal' build.",
```

Reply:
I've renamed it and updated the description so it's more clear. I'm not sure if CMakeUserPresets allows arbitrary fields since it's validated against a schema.